src/framebus.ts

```diff
@@ -17,10 +18,12 @@ type Listener = {
   originalHandler: FramebusOnHandler;
 };
 type VerifyDomainMethod = (domain: string) => boolean;
+type IFrameOrWindowList = Array<HTMLIFrameElement | Window>;
```

Comment:
```suggestion
// this is a mixed type so that users can add iframes to the array
// before they have been added to the DOM (in which case, they
// would not have a contentWindow yet). When accessing these
// windows in Framebus, the targetFramesAsWindows private 
// method should be used
type IFrameOrWindowList = Array<HTMLIFrameElement | Window>;
```